core/domain/skill_services.py

```diff
@@ -329,3 +329,71 @@ def save_skill_summary(skill_summary):
     )
 
     skill_summary_model.put()
+
+
+def create_skill_mastery(user_id, skill_id, degree_of_mastery):
+    """Creates and stores skill mastery of a user.
+
+    Args:
+        skill_id: str. The unique id of the skill.
+        user_id: str. The user id of the logged in user.
+        degree_of_mastery: float. The language code of the skill.
+    """
+    skill_mastery_model = skill_models.SkillMasteryModel(
```

Comment:
Have an id field here that can be used for searching, probably of the form USER_ID-SKILL_ID or something like that, so that it holds both information.